.circleci/config.yml

```diff
@@ -635,6 +658,8 @@ workflows:
       - devhub
       - docs
       - main
+      # Uncomment if you want to test the docker build
+      - build-image
```

Comment:
it's already uncommented though. - is this comment accurate?

Reply:
<img width="631" alt="image" src="https://github.com/mozilla/addons-server/assets/19595165/6b9a4a90-a6d5-4aab-9add-2f6ded2abb6a">

Reply:
Wanted to confirm it actually works. I'll remove the commit reverting back to commented out before merging.